h2o-docs/src/product/data-science/algo-params/custom_metric_func.rst

```diff
@@ -3,7 +3,7 @@
 ``custom_metric_func``
 ----------------------
 
-- Available in: GBM, DRF, Deeplearning, Stacked Ensembles, GLM
+- Available in: GAM, GBM, GLM, DRF, Deeplearning, Stacked Ensembles
```

Comment:
@wendycwong could you confirm that GLM and GAM support `custom_metric_func`? From the code it looks like it should be supported.

Reply:
Yuliia added support for custom_metric_func for GLM.  GAM does not support custom_metric_func yet.

Reply:
Okay, there is no test on it so I will treat it as it is not there.  Opened an issue here to make sure it works: https://github.com/h2oai/h2o-3/issues/16006 .  For now please treat it as it is not supported.